bool is_estimated 
float64 x
float64 y
float64 z
float64 bearing
float64 timestamp